### PR TITLE
add-member application event

### PR DESCRIPTION
**Pull request by opqdonut**
for #609

### CODE REVIEW

File: src/clj/rems/api/schema.clj [hunk 1]
```diff
@@ -43,7 +43,8 @@
    :round s/Num
    :event s/Str
    :comment (s/maybe s/Str)
-   :time DateTime})
+   :time DateTime
+   :eventdata s/Any})
```
Comment (opqdonut) [inline src/clj/rems/api/schema.clj]: I'm not so sure about this...

Comment (luontola) [reply src/clj/rems/api/schema.clj]: `{s/Keyword s/Any}` perhaps?

**Comment by opqdonut** [comment]
will rebase on top of master

File: src/clj/rems/api/schema.clj (hunk 1 above)
Comment (opqdonut) [reply src/clj/rems/api/schema.clj]: I think an opaque s/Any is pretty clear. The contents of eventdata is up to the event implementations. I'm not sure whether we should even publish it via the api.

**Pull request merged**